lms/djangoapps/instructor_task/tests/test_tasks.py

```diff
@@ -334,6 +354,25 @@ def test_overriding_success(self):
             action_name='overridden'
         )
 
+    def test_overriding_success_with_no_state(self):
+        """
+        Tests that score override is successful for a learner even he has no state
```

Comment:
`Tests that score override is successful for a learner when they have no state.`